scripts/testenv/testenvSettings.sh

```diff
@@ -1,18 +1,25 @@
 #!/usr/bin/env bash
 set +x
+set -e
 
-if [[ $USE_TESTENV_PROPERTIES == true ]]; then
+if [ $USE_TESTENV_PROPERTIES = true ]; then
     testenv_file="./testenv/testenv.properties"
-    if [[ "$PLATFORM" == *"zos"* ]]; then
-        testenv_file="./testenv/testenv_zos.properties"
-    fi
-    if [[ "$PLATFORM" == *"arm"* ]] && [[ "$JDK_VERSION" == "8" ]]; then
-        testenv_file="./testenv/testenv_arm32.properties"
-    fi
+    case "$PLATFORM" in
+        *zos*)
+            testenv_file="./testenv/testenv_zos.properties"
+            ;;
+        *arm*)
+            case "$JDK_VERSION" in
+                "8")
+                    testenv_file="./testenv/testenv_arm32.properties"
+                    ;;
+            esac
+            ;;
+    esac
```

Comment:
This isn't what I meant by "two sequential case statements" (this is just more verbose than the earlier version).
What I meant was:
```
    testenv_file="./testenv/testenv.properties"
    case "$PLATFORM" in
        *zos*)
            testenv_file="./testenv/testenv_zos.properties"
            ;;
        *)
            ;;
    esac
    case "$PLATFORM" in
        *arm*)
            if [ "$JDK_VERSION" = "8" ]; then
                testenv_file="./testenv/testenv_arm32.properties"
            fi
            ;;
        *)
            ;;
    esac
```
Again, this is only relevant for platforms matching `*arm*zos*` or `*zos*arm*` (which may never materialize).